RecoParticleFlow/PFClusterProducer/plugins/PFCTRecHitProducer.cc

```diff
@@ -856,36 +857,25 @@ PFCTRecHitProducer::createHcalRecHit( const DetId& detid,
       <<layer<<endl;
     return 0;
   }
-  
-  const GlobalPoint& position = thisCell->getPosition();
-  
-  double depth_correction = 0.;
+    
+  // double depth_correction = 0.;
   switch ( layer ) { 
   case PFLayer::HF_EM:
-    depth_correction = position.z() > 0. ? EM_Depth_ : -EM_Depth_;
-    break;
+    // depth_correction = position.z() > 0. ? EM_Depth_ : -EM_Depth_;
```

Comment:
this  could be removed instead of commenting out.
The feature is implemented in IdealZPrism already